docs/conf.py

```diff
@@ -219,3 +222,12 @@ def setup(app):
 
 # Prevent alphabetic sorting of (@data)class attributes/methods
 autodoc_member_order = 'bysource'
+
+# Nice formatting of model-specific input parameters
+napoleon_custom_sections = [
+    ('hype', 'params_style'),
+    ('lisflood', 'params_style'),
+    ('marrmot', 'params_style'),
+    ('pcrglobwb', 'params_style'),
+    ('wflow', 'params_style'),
+]
```

Comment:
I like how it looks at https://ewatercycle--98.org.readthedocs.build/en/98/apidocs/ewatercycle.forcing.html#ewatercycle.forcing.generate
, but having an additional place to edit when adding a model is something I want to avoid.  Can we make it dynamic?
```suggestion
from ewatercycle.forcing import FORCING_CLASSES
napoleon_custom_sections = [(m, 'params_style') for m in FORCING_CLASSES.keys()]
```

Reply:
It appears spinx doesn't like this import

Reply:
Hmmpf, tried to use .readthedocs.yml with conda install (17addd6e1fe7e219ae0338cefcdae76fc0105da6), but [build failed with `Command killed due to excessive memory consumption`](https://readthedocs.org/projects/ewatercycle/builds/13967669/)

I reverted it back to having a hardcoded list and updated #48